src/main/java/uk/gov/digital/ho/hocs/info/api/CountryResource.java

```diff
@@ -27,6 +33,34 @@ public CountryResource(CountryService countryService) {
         return ResponseEntity.ok(countrys.stream().map(Country::getName).collect(Collectors.toSet()));
     }
 
+    @GetMapping(value = "/countries/local", produces = APPLICATION_JSON_UTF8_VALUE)
+    public ResponseEntity<?> getAllActiveCountries() {
+
+        log.info("getAllActiveCountries - local resource");
+        File countries = null;
+        try {
+            countries = new ClassPathResource("countries-list.json").getFile();
+        } catch (IOException e) {
+            log.info("local countries file not found.");
+            ResponseEntity.badRequest();
+        }
+        return ResponseEntity.ok().body(new FileSystemResource(countries));
+    }
+
+    @GetMapping(value = "/territories/local", produces = APPLICATION_JSON_UTF8_VALUE)
+    public ResponseEntity<?> getAllActiveTerritories() {
+
+        log.info("getAllActiveTerritories - local resource");
+        File territories = null;
+        try {
+            territories = new ClassPathResource("territories-list.json").getFile();
+        } catch (IOException e) {
+            log.info("local territories file not found.");
```

Comment:
should this be higher than `info`?

Reply:
it is DECS convention, what do you think would be appropriate.

Reply:
The failures are now logged at error level. I should point out that there was no logging at all in this class previously.

Reply:
We do need a review of the the logging levels. I'm happy with this been logged as an error - to which it is.